ark/phenotyping/som_utils.py

```diff
@@ -13,26 +13,40 @@
 from ark.utils import misc_utils
 
 
-def create_pixel_matrix(img_xr, seg_labels, fovs=None, channels=None, blur_factor=2):
+def create_pixel_matrix(img_xr, seg_labels, base_dir,
+                        pre_name='pixel_mat_preprocessed.hdf5',
+                        sub_name='pixel_mat_subsetted.hdf5', fovs=None, channels=None,
```

Comment:
We're curently passing channels to both this fucntion and som_train.R. We only need to subset channels once, so we should pick where we want this to happen. 

It would be easier if we just passed the channels subset command to the train/run function. That way, the data only has to be preprocessed once, and if people want to try out different channels to use for the SOM, they can just switch the argument. However, if preprocessing takes a super long time, then preprocessing all 40 channels (vs maybe like 5-10 which are required) would have a large computational cost. Based on your benchmarking, we can make a decision.

Reply:
Now that benchmarking has been complete, I think it makes the most sense to preprocess all the channels and just pass the channel subset to the train/cluster scripts.

Reply:
Sounds good